src/promptflow/promptflow/_sdk/_service/apis/connection.py

```diff
@@ -33,6 +33,10 @@
 # Response model of connection operation
 dict_field = api.schema_model("ConnectionDict", {"additionalProperties": True, "type": "object"})
 
+# Define get connection request parsing
+get_connection_parser = api.parser()
+get_connection_parser.add_argument("with_secrets", type=bool, default=False, location="args", required=False)
```

Comment:
I think we should add a new api instead of just a argument.

Reply:
add a new api for connection with secret